invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -30,39 +41,88 @@ export default class DocumentMetadata extends Component {
     );
   }
 
-  render() {
-    const document = this.props.documentDetails;
-    const rows = [
-      { name: 'Title', value: document.metadata.title },
-      { name: 'Authors', value: document.metadata.authors },
-    ];
-    if (!_isEmpty(document.metadata.keywords)) {
-      rows.push({
-        name: 'Keywords',
-        value: this._renderKeywords(document.metadata.keywords),
-      });
-    }
-    const header = (
+  handleOnRefClick(loanPid) {
+    const navUrl = BackOfficeRoutes.loanDetailsFor(loanPid);
+    window.open(navUrl, `_loan_${loanPid}`);
+  }
+
+  createRefProps(documentPid) {
+    const loanStates = invenioConfig.circulation.loanActiveStates;
+    loanStates.push('PENDING');
```

Comment:
I think you are missing `CIRCULATION_STATES_LOAN_ACTIVE` to be align with backend.

Reply:
Actually I am using the states from the backend, but `PENDING` is not in them so I push it.